Resources/Prototypes/Catalog/VendingMachines/Inventories/salvage.yml

```
@@ -14,3 +14,4 @@
     SeismicCharge: 2
     FultonBeacon: 1
     Fulton: 2
+    ClothingOuterHardsuitSpatio: 2
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)